custom_dc/index.md

```diff
@@ -55,22 +55,19 @@ For the following use cases, a custom Data Commons instance is not necessary:
 
 ![setup1](/assets/images/custom_dc/customdc_setup1.png){: height="450" }
 
-The custom Data Commons Docker container consists of the following components:
+A custom Data Commons instance uses custom data that you provide as raw CSV files. An importer script converts the CSV data into the Data Commons format and stores this in a SQL database. For local development, we provide a lightweight, open-source [SQLite](http://sqlite.org) database; for production, we recommend that you use [Google Cloud SQL](https://cloud.google.com/sql/).
 
-- A [Nginx reverse proxy server](https://www.nginx.com/resources/glossary/reverse-proxy-server/), which routes incoming requests to the web or API server
-- A Python-Flask web server, which handles interactive requests from users
-- An Python-Flask NL server, for generating embeddings and serving natural language queries
-- A Go Mixer, also known as the API server, which serves programmatic requests using Data Commons APIs. The SQL query engine is built into the Mixer, which sends queries to both the local and remote data stores to find the right data. If the Mixer determines that it cannot fully resolve a user query from the custom data, it will make an REST API call, as an anonymous "user" to the base Data Commons Mixer and data.
+In addition to the data, a custom Data Commons instance consists of two Docker containers: one with the core services that serve the data and website; and one with additional utilities for managing and loading custom data and embeddings used for natural-language processing. 
 
-In addition to the Docker container, a custom Data Commons instance uses custom data that you provide as raw CSV files. The web server calls an importer script that converts the CSV data into the Data Commons format and stores this in a SQL database. For local development, we provide a lightweight, open-source [SQLite](http://sqlite.org) database; for production, we recommend that you use [Google Cloud SQL](https://cloud.google.com/sql/).
+Details about the components that make up the containers are provided in the [Getting started](quickstart.md) guide.
 
 ## Requirements and cost
 
-A custom Data Commons site runs in a Docker container on Google Cloud Platform (GCP). You will need the following:
+A custom Data Commons site runs in a Docker container on Google Cloud Platform (GCP), using Google Cloud Run a serverless solution that is by far the simplest and least expensive option, providing auto-scaling and other benefits. You will need the following:
```

Comment:
Perhaps leave this out? "that is by far the simplest and least expensive option"

Reply:
OK.